### PR TITLE
feat: adding rename-packages script

### PR DESCRIPTION
**Pull request by imCorfitz**
This PR addresses the feature request: #63

Screenshot 2024-05-28 at 13 12 14

Few things to address:
- [x] Update documentation
- [x] Add rename root package option
- [x] Discuss the proper line argument name
- [x] Explain to @imCorfitz what "nukes itself after you run it." means

### CODE REVIEW

**Comment by imCorfitz** [comment]
Maybe we could extend on this script a little bit. Might be of interest to also change the author for all packages. Maybe even an ignore patterns argument to exclude certain packages.

File: scripts/set-author.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+// ------------------------------------------------------------------
+
+// Parse command line arguments
+const argv = yargs(hideBin(process.argv))
+  .option('author', {
+    alias: 'a',
+    type: 'string',
+    demandOption: true,
+    description: 'The new version for the packages',
```
Comment (ixahmedxi) [inline scripts/set-author.ts:16]: typo

Comment (ixahmedxi) [reply scripts/set-author.ts:16]: not typo but incorrect description

Comment (imCorfitz) [reply scripts/set-author.ts]: Well spotted. Now fixed

**Review by ixahmedxi — DISMISSED** [review]
Lovely, amazing work!

**Review by ixahmedxi — APPROVED** [review]

**Pull request merged**